legal-api/src/legal_api/core/filing.py

```diff
@@ -405,6 +405,7 @@ def common_ledger_items(business_identifier: str, filing_storage: FilingStorage)
         filing = Filing()
         filing._storage = filing_storage  # pylint: disable=protected-access
         return {
+            'displayLedger': filing_storage.filing_type != Filing.FilingTypes.ADMIN_FREEZE,
```

Comment:
I was thinking an approach that would allow us to add filings that should not be displayed in the ledger.

Maybe create a list constant that contains filing types that shouldn't be displayed on the ledger.